src/util.ts

```diff
@@ -0,0 +1,40 @@
+import Context from './context'
+
+export const env = process.env
+
+export function assert<C extends Context, T extends keyof C>(
+  ctx: C,
+  value: T,
+  method: string
+): asserts ctx is C & { [k in T]: NonNullable<C[T]> } {
+  if (ctx[value] === undefined) {
+    throw new TypeError(
+      `Telegraf: "${method}" isn't available for "${ctx.updateType}"`
+    )
+  }
+}
```

Comment:
Why not just make `Context::assert` `@internal public` and consistently use it everywhere?

Reply:
What's `@internal`?

Reply:
https://www.typescriptlang.org/tsconfig#stripInternal